src/main/java/com/gdsc/studyjamapi/common/Constants.java

```diff
@@ -18,6 +18,7 @@ public static class ErrorMessage {
     public static final String SHORT_PASSWORD =
         "The length of the password must be at least 8 characters.";
     public static final String EMAIL_ALREADY_EXISTS = "This email already exists.";
+    public static final String EMAIL_NOT_FOUND = "Not found this email.";
     public static final String NOT_FOUND_MESSAGE_TEMPLATE = "%s not found";
```

Comment:
I already created the `NOT_FOUND_MESSAGE_TEMPLATE` constant below, you should use it to reduce duplicated code and keep the consistency.

Reply:
I don't know we can reuse it, sorry 😢